pandas/core/generic.py

```diff
@@ -777,6 +777,7 @@ def swapaxes(
         return self._constructor(
             new_values,
             *new_axes,
+            copy=False,
```

Comment:
This can be passed here because in case of CoW, the single block case is already handled above? So if we end up here and use CoW, we always have multiple blocks and thus already a copy?

I would maybe add a brief comment mentioning that, maybe like

```
# ...
copy = False
```

to have the comment together with the value, and then passing `copy` to the constructor.

Reply:
Yep exactly. Added a comment